test/e2e/tests/confirmations/transactions/erc721-approve-redesign.spec.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires */
+import { veryLargeDelayMs, WINDOW_TITLES } from '../../../helpers';
+import { Driver } from '../../../webdriver/driver';
+import { scrollAndConfirmAndAssertConfirm } from '../helpers';
+import {
+  openDAppWithContract,
+  TestSuiteArguments,
+  toggleAdvancedDetails,
+} from './shared';
+
+const {
+  defaultGanacheOptions,
+  defaultGanacheOptionsForType2Transactions,
+  withFixtures,
+} = require('../../../helpers');
+const FixtureBuilder = require('../../../fixture-builder');
+const { SMART_CONTRACTS } = require('../../../seeder/smart-contracts');
+
+describe('Confirmation Redesign ERC721 Approve Component', function () {
+  const smartContract = SMART_CONTRACTS.NFTS;
+
+  describe('Submit an Approve transaction @no-mmi', function () {
+    it('Sends a contract interaction type 0 transaction (Legacy)', async function () {
```

Comment:
Should these `it` blocks omit the contract interaction and just be `Sends a type 0 transaction (Legacy)` for example?